packages/chaire-lib-backend/src/tasks/dataImport/OsmDataPreparationResidential.ts

```diff
@@ -156,7 +158,8 @@ export default class OsmDataPreparationResidential {
 
         // Do we have a home or main entrance
         const entrances = getEntrancesForBuilding(building, homeObject, osmRawData, {
-            entranceTypes: ['main', 'home']
+            entranceTypes: ['main', 'home'],
+            skipInsideNodesOnUndefinedError: this._continueOnMissingGeojson
```

Comment:
Now you set this other variable on this new flag, this is confusing without explanation.